src/variables.py

```diff
@@ -16,6 +16,7 @@
 # - App specific -
 LIDO_LOCATOR_ADDRESS = os.getenv('LIDO_LOCATOR_ADDRESS')
 FINALIZATION_BATCH_MAX_REQUEST_COUNT = os.getenv('FINALIZATION_BATCH_MAX_REQUEST_COUNT', 1000)
+ALLOW_NEGATIVE_REBASE_REPORTING = bool(int(os.getenv('ALLOW_NEGATIVE_REBASE_REPORTING', 0)))
```

Comment:
Maybe better this way?
```suggestion
ALLOW_NEGATIVE_REBASE_REPORTING = os.getenv('ALLOW_NEGATIVE_REBASE_REPORTING', 'False') == 'True'
```

Reply:
yea, agree